docs/run-platform-server/monitoring.mdx

```diff
@@ -25,6 +59,8 @@ Build queries that highlight performance of a given Horizon deployment. Refer to
 - Average ingestion time of a ledger.
 - Average ingestion time of a transaction.
 
+Choose the (revisions tab)[https://grafana.com/grafana/dashboards/13793-stellar-horizon/?tab=revisions], and download the dashboard source file to have access to the grafana dashboard source code and metrics queries that build each panel in dashboards.
+
 ### Alerts
 
 Once queries are developed on a Grafana dashboard, it enables convenient follow-on step to add [alert rules](https://grafana.com/docs/grafana/latest/alerting/alerting-rules/create-grafana-managed-rule/) based on specific queries to trigger notifications when thresholds are exceeded.
```

Comment:
Can we add some more examples in the table?

Reply:
yes, added one more for a wide area related to http errors - https://github.com/stellar/stellar-docs/pull/198/commits/9fcae17acf12f491eff3a98a90700f5e74278d79